omnigibson/robots/tiago.py

```diff
@@ -475,29 +462,16 @@ def base_idx(self):
     def trunk_control_idx(self):
         """
         Returns:
-            n-array: Indices in low-level control vector corresponding to trunk joint.
+            n-array: Indices in low-level control vector corresponding to trunk joints.
         """
-        return np.array([6])
-
-    @property
-    def camera_control_idx(self):
-        """
-        Returns:
-            n-array: Indices in low-level control vector corresponding to [tilt, pan] camera joints.
-        """
-        return np.array([9, 12])
+        return np.array([list(self.joints.keys()).index(name) for name in self.trunk_joint_names])
 
     @property
     def arm_control_idx(self):
-        return {
-            "left": np.array([7, 10, 13, 15, 17, 19, 21]),
-            "right": np.array([8, 11, 14, 16, 18, 20, 22]),
-            "combined": np.array([7, 8, 10, 11, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]),
-        }
-
-    @property
-    def gripper_control_idx(self):
-        return {"left": np.array([23, 24]), "right": np.array([25, 26])}
+        # Add combined entry
+        idxs = super().arm_control_idx
+        idxs["combined"] = np.sort(np.concatenate([val for val in idxs.values()]))
+        return idxs
```

Comment:
Is this change intended? What happened to the "left" and the "right"?

Reply:
It's all automatically inferred now based on arm_joint_names!